src/modules/nRF24/nRF24.h

```diff
@@ -208,8 +215,12 @@ class nRF24: public PhysicalLayer {
 
       \returns \ref status_codes
     */
-    int16_t begin(int16_t freq = 2400, int16_t dataRate = 1000, int8_t power = -12, uint8_t addrWidth = 5);
-
+    //int16_t begin(int16_t freq = 2400, int16_t dataRate = 1000, int8_t power = -12, uint8_t addrWidth = 5);
+      int16_t begin(
+      int16_t freq = RADIOLIB_NRF24_DEFAULT_FREQ,
```

Comment:
Also strange indentation - I think all the lines containing parameters should be indented by two spaces.